metastats-domain/src/main/groovy/life/qbic/metastats/datamodel/MetaStatsExperiment.groovy

```diff
@@ -5,12 +5,20 @@ class MetaStatsExperiment {
     String type
     List<String> samples
     Map<String,String> properties
```

Comment:
make them `final` when you don't want a setter. This is groovy "magic"

Reply:
0101820